homework/src/app.js

```diff
@@ -1,11 +1,254 @@
 {
   /* Your homework goes here */
-  
   /* STRINGS */
-
   /* ARRAYS */
-
   /* OPERATORS */
-
   /* QUESTIONS FROM HYF REPOS */
-}
\ No newline at end of file
+}
+
+// STRINGS
+// Strings Question 1
+
+//  Consider the following string:
+
+const myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+// Q: Add the string to your file and log it.
+
+// Solution
+
+console.log(myString);
+
+// Log the length of myString.
+
+// Solution : we use length property for the solution.
+
+console.log(myString.length);
+
+// Q:The commas make that the sentence is quite hard to read. Find a way to remove the commas from the string and replace them with spaces.
+
+// Solution
+
+const myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+let regex = /,/gi;
+
+console.log(myString.replace(regex, ' '));
+
+// Strings Question 2:
+
+// Consider the following string:
+
+const str = 'dlroW olleH';
+
+// Q:The string doesn't make any sense. Find a way to reverse this string so that it becomes readable.
+
+// first i want to turn the string to an array, for that purpose I used split property
+let myNewArray = str.split('');
+
+console.log(myNewArray);
+
+// then i want to use reverse property to make the it readable and meaningful
+let mySecondArray = myNewArray.reverse();
+
+console.log(mySecondArray);
+
+// Now I want to get rid off the commas;
+
+console.log(mySecondArray.replace(regex, ' '));
```

Comment:
Aahaan. Couple things here:
1. Firstly, it would be really nice if you create a function for this question. This is because functions encapsulate logic and makes your code reusable. Thus, you would be able to use that function for not just `'dlroW olleH'` string but for any kind of strings.
2. Secondly, try to explore the `join` method which is quite like an inverse of the `split` method on strings.

What do you think of this? 😉 
```
function reverseString(input) {
  return input.split('').reverse().join('');
}
```